neurox/interpretation/iou_probe.py

```diff
@@ -0,0 +1,31 @@
+from sklearn.metrics import average_precision_score
+import numpy as np
+def get_neuron_ordering(X_train, y_train, threshold):
```

Comment:
Does the ordering change significantly if threshold == 0? Can we have a "good" default value for `threshold`, or is it very dataset dependent?